configure.py

```diff
@@ -192,6 +194,8 @@ def _run_command(self, cmdline):
 profilers = ['gmon', 'pprof']
 parser.add_option('--bootstrap', action='store_true',
                   help='bootstrap a ninja binary from nothing')
+parser.add_option('--bootstrap_from_amalgamation', action='store_true',
+                  help='bootstrap a ninja binary from the amalgamation')
```

Comment:
I think if we think amalgamation is a good idea, we should just always do it and not put it behind an option.  Or decide to never do it for bootstrap.  But either way, no option.